product_docs/docs/pgd/5/quickstart/further_explore_replication.mdx

```diff
@@ -0,0 +1,211 @@
+---
+title: Exploring replication with PGD
+navTitle: Exploring replication
+deepToC: true
+---
+
+## Explore replication with PGD
+
+With the cluster up and running, it's worthwhile to run some basic checks to see how effectively it's replicating.
```

Comment:
Consider "useful" instead of "worthwhile." Or, "we recommend running..."  I think this is more precise.